gluten-core/src/main/scala/io/glutenproject/expression/ConverterUtils.scala

```diff
@@ -479,8 +479,8 @@ object ConverterUtils extends Logging {
   final val TAN = "tan"
   final val TANH = "tanh"
   final val BITWISE_NOT = "BitwiseNot"
-  final val BITWISE_AND = "BitwiseAnd"
-  final val BITWISE_OR = "BitwiseOr"
+  final val BITWISE_AND = "bitwise_and"
+  final val BITWISE_OR = "bitwise_or"
```

Comment:
why ? it's better use consistent name for gluten, BitwiseNot, BitwiseAnd, why for Velox only ?

Reply:
What's the name in substrait? We should use the one in substrait, if substrait doesn't have it yet, we should submit a PR to add it, following substrait's function name pattern.

Reply:
Got it. I just noted these functions are differently named in velox & CH. We have a pending PR to substrait. Let's follow the naming in substrait after that PR is merged. Thanks!
https://github.com/substrait-io/substrait/pull/370